web-dashboard/public/index.html

```
@@ -0,0 +1,221 @@
+
+    <div class="stats-row" id="quick-stats"></div>
+
+    <div class="main-grid">
+        <div class="command-center">
+            <div class="panel">
+                <div class="panel-header"><span class="panel-title">Command Center</span></div>
+                <div class="jd-input-section">
+                    <p style="font-family: var(--mono); font-size: 11px; color: var(--txt-muted); margin-bottom: 8px;">/ career-ops — auto-pipeline</p>
+                    <div class="jd-wrapper">
+                        <textarea id="jd-input" placeholder="Paste JD or URL..."></textarea>
+                        <button class="run-btn" onclick="runAutoPipeline()">Run ↗</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)